lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -5,20 +5,27 @@
 import type { APICallInfo } from './config';
 
 /**
- * Object representing a file or folder resource in the LMS's file storage.
+ * Object representing a file, folder or document stored by the LMS
  */
-export type File = {
-  // FIXME - This ought to be present on all file objects.
-  type?: 'File' | 'Folder';
+export type Resource = {
```

Comment:
Did the term "Resource" come from anywhere in particular? It's a reasonable choice, although I wonder if "Document" might be a slightly more obvious generalization of files / pages etc.

Reply:
Nop, nowhere specific. I just tried to think on a term that would cover Files and Pages.

I'm fine with using `Document` instead.

Reply:
I have replaced any reference to `resource` with `document`, including the API type, the List component (which is now called `DocumentList`, not `ResourceList`) and any prop or comment.